checks/docker/add_instead_of_copy.rego

```diff
@@ -24,6 +24,10 @@ get_add[output] {
 	args := concat(" ", add.Value)
 
 	not contains(args, ".tar")
+
+	cnt := count(copy.Value)
+	not (cnt == 3 && startswith(add.Value[0], "file:") || startswith(add.Value[0], "multi:") && add.Value[1] == "in")
+
```

Comment:
I don't have permissions to make changes to this branch.

Rego does not have the `&&` and `||` operators. All expressions in a rule are evaluated sequentially, which is equivalent to conjunction.

```suggestion
	not is_command_with_hash(add.Value, "file:")
	not is_command_with_hash(add.Value, "multi:")
```

```rego
is_command_with_hash(cmd, prefix) {
	count(cmd) == 3
	startswith(cmd[0], prefix)
	cmd[1] == "in"
}

```

You can also merge main into your branch to run tests.

Reply:
@candrews could you please update [this](https://github.com/aquasecurity/trivy-policies/pull/56) PR as well?

Reply:
Thank you for providing the ability to run tests, that's very helpful!

I've updated this MR with your suggestion.